serving/src/main/java/feast/serving/util/Metrics.java

```diff
@@ -27,7 +27,7 @@
           .name("request_latency_seconds")
           .subsystem("feast_serving")
           .help("Request latency in seconds")
-          .labelNames("method")
+          .labelNames("service", "method", "status_code")
```

Comment:
Why is service hardcoded here?

Reply:
Was following the standard here: https://github.com/gojek/feast/blob/master/core/src/main/java/feast/core/grpc/interceptors/MonitoringInterceptor.java#L48-L50

But it did lead to some awkward code like this https://github.com/gojek/feast/blob/7ef87257d5903a08b9eaa3573f95a822db8a4326/serving/src/main/java/feast/serving/service/RedisServingService.java#L274

Separating into a different metric would solve this. Will be reverting this and changing it to a counter instead. There are some manual instrumentation (`feast_serving_request_latency_seconds`) for private methods that I won't touch (`RedisServingService.sendAndProcessMultiGet`, `RedisServingService.isStale`) just in case people are filtering by that label.